metadata.yaml

```diff
@@ -52,6 +52,8 @@ requires:
   db:
     interface: mysql
     limit: 1
```

Comment:
Should we consider drop the support for the legacy mysql relation?

Reply:
I was thinking about deprecating it gradually like the nginx-ingress-integrator, perhaps setting a timeline/notice for deprecation?

Reply:
As per discussion, this will be another spec to discuss about the deprecation of db/mysql relation :)